src/components/questions/DateOfBirth.tsx

```diff
@@ -10,15 +10,6 @@ import { Questions }                 from '../lib/Questions';
 import { Steps }                     from '../lib/Steps';
```

Comment:
`import { capitalize }                from 'lodash';` is unused.

Also, the `../lib` vs `../../lib` Can be a bit confusing at times.

Reply:
No need to fix this for this checkin. This is purely an observation.